samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,280 @@
+import os
+import copy
+import shutil
+import numpy as np
+import cv2 as cv
+import argparse
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--input', help='Path to input video file.')
```

Comment:
Please align with other samples (add `-i` short flag and let use camera or video file).